test/unit-test/FreeRTOS_DNS_Cache/FreeRTOS_DNS_Cache_utest.c

```diff
@@ -43,8 +43,7 @@
 #include "mock_FreeRTOS_DNS_Parser.h"
 #include "mock_FreeRTOS_DNS_Networking.h"
 #include "mock_NetworkBufferManagement.h"
-#include "FreeRTOS_DNS.h"
-
+#include "mock_FreeRTOS_DNS.h"
```

Comment:
This should be #include "FreeRTOS_DNS.h"

Reply:
If that's not the target file to test, we can include mock header instead.

Reply:
But it should incldue FreeRTOS_DNS_Cache.h here.

Reply:
I was under the impression that we have one header for all DNS files, but thats not the case. 
In this case we do not need xStub_pxNew_AddrInfo
We can just use the mocked function.